frontend/src/pages/projects/notebook/NotebookImagePackageDetails.tsx

```diff
@@ -28,9 +28,9 @@ const NotebookImagePackageDetails: React.FC<NotebookPackageDetailsProps> = ({
         <DescriptionListTerm>{title || 'Packages'}</DescriptionListTerm>
         <DescriptionListDescription>
           {dependencies.map(getNameVersionString).map((pkg) => (
-            <p key={pkg}>
-              <Truncate content={pkg} />
-            </p>
+            <div key={pkg}>
```

Comment:
Why the change from a paragraph block?

Reply:
hmm, good catch, I might do some tests that time and forgot to change it back. Will revert this.